ister_gui.py

```diff
@@ -2733,7 +2742,9 @@ def __init__(self, *args, **kwargs):
     def _init_actions(self):
         # auto install assumed
         keyboard_select = KeyboardSelection()
-        network_requirements = NetworkRequirements(1, 6)
+        network_requirements = NetworkRequirements(1, 6,
+                                                   self.args['contenturl'],
+                                                   self.args['versionurl'])
```

Comment:
If you look on line 2722, self._init_actions() is called before self.args is set to kwargs on line 2733. Using self.args in _init_actions results in an `'Installation' object has no attribute 'args'` error.